.gitignore

```diff
@@ -1,3 +1,11 @@
+# dependency management
+poetry.lock
```

Comment:
should describe why it's gitignore'd in a comment (multiple versions). not currently using multiple versions bc Poetry bugs, but would prefer to (vs. manual `pip install`s in Travis config) once those are fixed.

Reply:
done